client/ayon_motionbuilder/api/pipeline.py

```diff
@@ -125,20 +128,18 @@ def ls():
     containers = []
     for obj_sets in FBSystem().Scene.Sets:
         for prop in obj_sets.PropertyList:
-            if prop.AsString() in {
-                AYON_CONTAINER_ID, AVALON_CONTAINER_ID
-                }:
+            if prop.GetName() == "containers":
                     containers.append(obj_sets)
 
     for container in sorted(containers, key=attrgetter("Name")):
-        yield parse_container(container)
+        yield(parse_container(container))
```

Comment:
Only this one is weird. It should be just `yield parse_container(container)`

Reply:
Resolved with this branch here(and already merged): https://github.com/ynput/ayon-motionbuilder/pull/6